tests/NpgsqlConnectionTests.fs

```diff
@@ -580,6 +582,19 @@ let ``Tuples command prepared``() =
 
     Assert.True(isStatementPrepared conn)
 
+[<Fact>]
```

Comment:
```suggestion
[<Fact>]
let ``Record type reused regardless of column order`` =
    use cmd1 = DvdRentalWithTypeReuse.CreateCommand<"select film_id, rating from film", SingleRow = true>(dvdRental)
    use cmd2 = DvdRentalWithTypeReuse.CreateCommand<"select rating, film_id from film", SingleRow = true>(dvdRental)
```

@daz10000, runtime column verification (including column order) is still performed on both of these commands. I'm merely sorting the columns in design time so that they can share the same provided type. I think it's preferable to the commands generating 2 distinct types in this case:

```fsharp
DvdRentalWithTypeReuse.``film_id:Int32, rating:Option<public.mpaa_rating>``
DvdRentalWithTypeReuse.``rating:Option<public.mpaa_rating>, film_id:Int32``
```